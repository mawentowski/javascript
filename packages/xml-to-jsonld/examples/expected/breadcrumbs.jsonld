{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position": 1,
      "name": "Home",
      "item": "/"
    },
    {
      "@type": "ListItem",
      "position": 2,
      "name": "Shop",
      "item": "/shop"
    },
    {
      "@type": "ListItem",
      "position": 3,
      "name": "Widget",
      "item": "/shop/widget"
    }
  ]
}