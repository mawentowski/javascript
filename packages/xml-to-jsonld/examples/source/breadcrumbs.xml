<!-- <?xml-model href="../packages/schemas/rnc/site-seo.rnc"
            type="application/relax-ng-compact-syntax"?> -->

<BreadcrumbList>
  <itemListElement>
    <ListItem>
      <position>1</position>
      <name>Home</name>
      <item>/</item>
    </ListItem>
    <ListItem>
      <position>2</position>
      <name>Shop</name>
      <item>/shop</item>
    </ListItem>
    <ListItem>
      <position>3</position>
      <name>Widget</name>
      <item>/shop/widget</item>
    </ListItem>
  </itemListElement>
</BreadcrumbList>