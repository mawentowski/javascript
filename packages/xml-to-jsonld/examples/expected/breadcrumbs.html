<nav aria-label="Breadcrumb"><ol><li><a href="/">Home</a></li><li><a href="/shop">Shop</a></li><li><a href="/shop/widget">Widget</a></li></ol></nav>